kf_lib_data_ingest/etl/transform/transform.py

```diff
@@ -82,18 +82,38 @@ def _read_output(self):
 
     def _write_output(self, output):
         """
-        Write output of transform stage to JSON file
+        Write output of transform stage to JSON files and tab separated value
+        files. TSV files are stored in a tsv folder within the stage's output
+        dir.
 
         :param output: output created by TransformStage._run
         :type output: a dict of pandas.DataFrames
         """
         assert_safe_type(output, dict)
         assert_all_safe_type(output.values(), list)
+
         paths = []
         for key, data in output.items():
-            fp = os.path.join(self.stage_cache_dir, key + '.json')
+            # JSON output
+            fp = os.path.join(self.stage_cache_dir, key) + '.json'
```

Comment:
you made a new dir for tsvs but not from jsons?

Reply:
Yea bc the tsv files are just an alternate form of the stage's serialized output. The files in the top level stage cache dir are the de facto serialized output so I didn't want to bury them in another dir